File: Python/math/eq.py
#a[0]*x+a[1]*y+a[2]=0; b[0]*x+b[1]*y+b[2]=0
def eq1_2(a,b):
	t = v3_pr(a,b)
	return [t[0]/t[2], t[1]/t[2]]

File: JavaScript/math/eq.js
//a.x*r1+a.y*r2+a.z=0; b.x*r1+b.y*r2+b.z=0
function eq1_2(a,b){
	let q = vt3.pr_v3(a,b);
	return new vt2(q.x/q.z, q.y/q.z);
}
